iot-management-device/device/src/test/kotlin/com/example/iotmanagementdevice/device/application/mapper/DeviceUpdateEventMapperTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.iotmanagementdevice.device.application.mapper
+
+import com.example.iotmanagementdevice.device.DeviceFixture.createDevice
+import com.example.iotmanagementdevice.device.DeviceFixture.deviceUpdatedEvent
+import com.example.iotmanagementdevice.device.application.mapper.impl.DeviceUpdateEventMapperImpl
```

Comment:
`application.mapper.impl` looks wierd to me. Seems, we should change the package